### PR TITLE
Fixed menu order and added missing translation

### PR DESCRIPTION
**Pull request by m-kharbat**
@chaudum please review

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,11 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+2016/09/01 0.19.1
+=================
```
Comment (bogensberger) [inline CHANGES.txt]: remove, it's still unreleased

File: CHANGES.txt [hunk 2]
```diff
@@ -5,6 +5,11 @@ Changes for Crate Admin Interface
...
+ - Fix: fixed menu order.
```
Comment (bogensberger) [inline CHANGES.txt]: add a line break

File: app/i18n/de.json [hunk 3]
```diff
@@ -23,7 +23,8 @@
       "OVERVIEW": "Übersicht",
       "CONSOLE": "Konsole",
       "TABLE": "Tabelle",
-      "CLUSTER": "Cluster"
+      "CLUSTER": "Cluster",
+      "GET_STARTED":"Loslegen"
```
Comment (chaudum) [inline app/i18n/de.json]: Is it somehow possible that the tutorial plugin provides its own i18n json file?
Not sure whether it is really doable, though.

File: app/plugins/tutorial/tutorial.js [hunk 4]
```diff
@@ -145,4 +144,11 @@ tutorialPlugin.run(function($window, $location, NavigationService) {
       localStorage.setItem('crate.start_twitter', "true");
       $window.location.href = path + '#/tutorial';
     }
+
+      // Update Navbar Elements if Language is Changed
```
Comment (chaudum) [inline app/plugins/tutorial/tutorial.js]: code style: indentation

File: app/scripts/controllers/common.js [hunk 5]
```diff
@@ -197,14 +196,13 @@ var commons = angular.module('common', ['stats', 'udc'])
...
-commons.run(function(NavigationService, $translate, $rootScope) {
+commons.run(function(NavigationService, $translate,  $filter, $rootScope) {
   // Initial translation of navigation bar items
-  $translate(['NAVIGATION.OVERVIEW','NAVIGATION.CONSOLE','NAVIGATION.TABLE','NAVIGATION.CLUSTER']).then(function(i18n) {
-    NavigationService.addNavBarElement("fa fa-th-large", i18n['NAVIGATION.OVERVIEW'], "/");
-    NavigationService.addNavBarElement("fa fa-code", i18n['NAVIGATION.CONSOLE'], "/console");
-    NavigationService.addNavBarElement("fa fa-table", i18n['NAVIGATION.TABLE'], "/tables");
-    NavigationService.addNavBarElement("fa fa-sitemap", i18n['NAVIGATION.CLUSTER'], "/nodes");
-  });
+    NavigationService.addNavBarElement("fa fa-th-large", $filter('translate', 'NAVIGATION.OVERVIEW'), "/");
```
Comment (chaudum) [inline app/scripts/controllers/common.js]: code style: indentation

File: app/scripts/controllers/common.js [hunk 6]
```diff
@@ -197,14 +196,13 @@ var commons = angular.module('common', ['stats', 'udc'])
...
+    NavigationService.addNavBarElement("fa fa-code", $filter('translate', 'NAVIGATION.CONSOLE'), "/console");
+    NavigationService.addNavBarElement("fa fa-table",$filter('translate', 'NAVIGATION.TABLE'), "/tables");
```
Comment (bogensberger) [inline app/scripts/controllers/common.js]: code style: space after  ','

**Comment by chaudum** [comment]
lgtm
please squash into single commit and merge
also don't forget to backport it to 0.19 branch. @bogensberger can show you how to do that

**Pull request merged**